concordium-consensus/src/Concordium/GlobalState/Persistent/CachedRef.hs

```diff
@@ -344,6 +344,33 @@ data HashedCachedRef' h c a
         hcrHash :: !h
     }
 
+-- |Migrate a 'HashedCachedRef'. This
```

Comment:
Documentation? Also, perhaps move this further down the file, after the definitions it uses.